JeffShepherd/fitness-tracker
# Feature/fetch api data

## What does this PR do?

- Add API call for user data
- Remove hardcoded user data file
- Update tests to use sample user data

## How should it be tested in the UI?

- Run 'npm start' and ensure application is populated with data on page load (no changes made in UI)
- Run 'npm test' and ensure all tests still pass
